server/gamenode/Lobby.ts

```diff
@@ -51,7 +53,55 @@ export class Lobby {
 
         if (this.game) {
             this.sendGameState(this.game);
+        } else {
+            this.sendDeckInfo();
+        }
+    }
+
+    private updateDeck(socket: Socket, ...args) {
+        const activeUser = this.users.find((u) => u.id === socket.user.username);
+        const userDeck = activeUser.deck.data;
+        const source = args[0]; // [<'Deck'|'Sideboard>'<cardID>]
+        const cardID = args[1];
+
+        Contract.assertTrue(source === 'Deck' || source === 'Sideboard');
```

Comment:
Since the default error message for this will just say "false condition", we should add an error description (second argument of assertTrue) that prints the actual value of `source` for debugging purposes